e2e/content-services/permissions-component.e2e.ts

```diff
@@ -210,4 +213,50 @@ describe('Permissions Component', function () {
 
     });
 
+    describe('Role Site Dropdown', function () {
+
+        beforeEach(async (done) => {
+            await alfrescoJsApi.login(fileOwnerUser.id, fileOwnerUser.password);
+            let siteName = `PUBLIC_TEST_SITE_${Util.generateRandomString(5)}`;
+            let folderName = `MEESEEKS_${Util.generateRandomString(5)}`;
+            let publicSiteBody = { visibility: 'PUBLIC', title: siteName };
+
+            publicSite = await alfrescoJsApi.core.sitesApi.createSite(publicSiteBody);
+
+            uploadedFolder = await uploadActions.createFolder(alfrescoJsApi, folderName, publicSite.entry.guid);
+
+            loginPage.loginToContentServicesUsingUserModel(fileOwnerUser);
```

Comment:
The lines above should probably be in beforeAll rather than beforeEach

Reply:
I think, it doesn't make sense, to put it all in beforeAll, as we are doing slightly different things for each of the tests and Describes beneath.